libminifi/include/utils/GeneralUtils.h

```diff
@@ -64,6 +64,7 @@ using std::exchange;
 #if __cplusplus < 201703L
 template<typename...>
 using void_t = void;
+
```

Comment:
Could you reset this file to master?
`git fetch apache && git checkout apache/master -- libminifi/include/utils/GeneralUtils.h`

Reply:
My central remote is called `central`. It won't work.

Jk, updated.